cfme/common/vm.py

```diff
@@ -717,6 +725,69 @@ def set_retirement_date(self, when, warn=None):
             sel.click(form_buttons.save)
 
 
+    def equal_drift_results(self, row_text, section, *indexes):
+        """ Compares drift analysis results of a row specified by it's title text
+
+        Args:
+            row_text: Title text of the row to compare
+            section: Accordion section where the change happened; this section must be activated
+            indexes: Indexes of results to compare starting with 0 for first row (latest result).
+                     Compares all available drifts, if left empty (default).
+
+        Note:
+            There have to be at least 2 drift results available for this to work.
+
+        Returns:
+            ``True`` if equal, ``False`` otherwise.
+        """
+        # mark by indexes or mark all
+        self.load_details(refresh=True)
+        sel.click(InfoBlock("Properties", "Drift History"))
+        if indexes:
+            drift_table.select_rows_by_indexes(*indexes)
+        else:
+            # We can't compare more than 10 drift results at once
+            # so when selecting all, we have to limit it to the latest 10
+            if len(list(drift_table.rows())) > 10:
+                drift_table.select_rows_by_indexes(*range(0, min(10, len)))
```

Comment:
If there are less than 10 rows, this same line will select all of them, correct? If so, you can remove the logic and just select min(10, len) in all cases.